ksp_plugin/manœuvre.hpp

```diff
@@ -0,0 +1,80 @@
+﻿#pragma once
+
+#include "geometry/named_quantities.hpp"
+#include "physics/trajectory.hpp"
+#include "quantities/named_quantities.hpp"
+
+namespace principia {
+
+using geometry::Instant;
+using geometry::Vector;
+using physics::Trajectory;
+using quantities::Force;
+using quantities::Mass;
+using quantities::Speed;
+using quantities::Time;
+using quantities::Variation;
+
+namespace ksp_plugin {
+
+// This class represents a constant-thrust inertial burn.
+template<typename Frame>
+class Manœuvre {
+ public:
+  Manœuvre(Force thrust,
+           Mass initial_mass,
+           Speed effective_exhaust_velocity,
+           Vector<double, Frame> direction);
+  ~Manœuvre() = default;
+
+  Force thrust() const;
```

Comment:
No, we can only do that for the ones that are stored, and that would break symmetry.